src/cli/call/procedure/Procedure.handler.ts

```diff
@@ -55,6 +55,7 @@ export default class ProcedureHandler extends DB2BaseHandler {
 
         // Return as an object when using --response-format-json
         params.response.data.setObj(response);
+        params.response.data.setExitCode(response.success ? 0 : 1);
```

Comment:
If every command handler extends `DB2BaseHandler`, could we move this line over there? 😋 

And maybe have a try/catch on the `DB2BaseHandler` to catch any imperative errors, set the exit code and then proceed to throw the error?

Technically speaking though, when an error is thrown, the exit code is should be automatically set to 1 (by the Imperative) 😋